File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {        
        int left = 0, maxProfit = 0;

        for(int r=1; r<prices.length; r++){
            if(prices[left] < prices[r]){
                maxProfit = Math.max(maxProfit, prices[r]-prices[left]);
            }else{
                left = r;
            }
        }        

        return maxProfit;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cs
public class Solution {
    public int LongestConsecutive(int[] nums) {
        if (nums.Length == 0)
            return 0;

        Array.Sort(nums);
        int left = 0, maxLength = 1, currentMax = 1;
        for (int i = 1; i < nums.Length; i++) {
            if (nums[i] == nums[i - 1])
                continue;

            if (nums[i] == (nums[i - 1] + 1)) {
                currentMax++;
            } else {
                maxLength = Math.Max(currentMax, maxLength);
                currentMax = 1;
            }
        }
        maxLength = Math.Max(currentMax, maxLength);
        return maxLength;
    }
}
